### DISCUSSION TITLE
请为发音评估功能的原理是什么/

### DISCUSSION THREAD

**Discussion by zhouartisan** [ZuodaoTech/everyone-can-use-english#289] [Q&A]
自己发音挺差的，断句和音调都把握不住，但是系统也会打90多分，不是太理解

**Comment by an-lee** [comment]
只能用来参考。

> “发音分数”：这个分数表示对发音质量的综合评估，包括 4 个子方面。 “阅读”和“口语”选项卡中都提供了这些分数，用于脚本化评估和未脚本化评估。
> 准确度分数：评估发音的正确性。
> 流畅性分数：衡量语音的流畅程度和自然程度。
> 完整性分数：反映正确发音的单词数。
> 韵律分数：评估对语调、节奏和重音的恰当运用。 引入了更多与韵律评估相关的错误类型，例如意外断句、缺少断句和语调单一。 与以前的引擎相比，这些错误类型提供了有关发音错误的更详细信息。

https://learn.microsoft.com/zh-cn/azure/ai-services/speech-service/pronunciation-assessment-tool?tabs=display
